utils/rpms/daos.spec

```diff
@@ -85,7 +85,7 @@ BuildRequires: libisa-l_crypto-devel
 BuildRequires: libisal-devel
 BuildRequires: libisal_crypto-devel
 %endif
-BuildRequires: daos-raft-devel = 0.8.1
+BuildRequires: daos-raft-devel = 0.9.0-1394.gc81505f%{?dist}%{?dist}
```

Comment:
[Annotation] Let's change this back to a single "dist" next time we update it (to point to a fixed raft-devel RPM).

Reply:
PR for that has landed, btw.